examples/tutorials/3d-perspective-image.py

```diff
@@ -4,6 +4,13 @@
 
 Create 3-D perspective image or surface mesh from a grid
 using :meth:`pygmt.Figure.grdview`.
+
+This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+To see the figures while using a Python script instead, use
+``fig.show(method="external)`` to display the figure in the default PDF viewer.
+
+To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+is the desired name and file extension for the saved figure.
```

Comment:
Perhaps looks better using a `note` directive?
![image](https://user-images.githubusercontent.com/3974108/107433058-d69c5200-6af6-11eb-90d9-47fe658bbbdd.png)


```suggestion
.. note::

    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
    To see the figures while using a Python script instead, use
    ``fig.show(method="external)`` to display the figure in the default PDF viewer.

    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
    is the desired name and file extension for the saved figure.
```

Reply:
I changed this in the tutorial files in dd3ab29